support-frontend/assets/components/productOption/productOption.jsx

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import cx from 'classnames';
+
+// components
+import AnchorButton, { type PropTypes } from 'components/button/anchorButton';
+import { defaultProps } from 'components/button/_sharedButton';
+
+// styles
+import './productOption.scss';
+
+type Props = {
+  children: Node,
+};
+
+// hocs
+const withProductOptionsStyle = WrappedComponent => (props: PropTypes) => (
+  <div className="product-option__button">
+    <WrappedComponent {...props} />
+  </div>
+);
+
+// presentation components
+export const Line = ({ showLine, style }: { showLine: boolean, style?: { [string]: string } }) =>
```

Comment:
I'm a bit worried that these names are very generic whereas their application is very specific (they are only used with the ProductOptions component), for instance we already have a Content element in layout.jsx which represents the content for the entire page.
Could you add some prefix onto them to make it clearer that they are subcomponents of the ProductOptions component?